pulp_container/tests/functional/api/test_push_signatures.py

```diff
@@ -0,0 +1,125 @@
+"""Tests that verify that an image signature can be pushed to Pulp."""
+import gnupg
+import base64
+import json
+import os
+import re
+import requests
+import subprocess
+import pytest
+
+from pulp_smash.pulp3.bindings import (
+    delete_orphans,
+    monitor_task,
+)
+from pulp_container.tests.functional.constants import REGISTRY_V2_REPO_PULP
+from pulp_container.tests.functional.utils import get_auth_for_url
+
+gpg = gnupg.GPG()
+
+@pytest.fixture
+def key_data():
```

Comment:
I've seen us using this key for tests: https://github.com/pulp/pulp-fixtures/raw/master/common/GPG-PRIVATE-KEY-pulp-qe

It's probably better than regenerating a new one all the time.

Reply:
Also this fixture is probably worth to be session scoped.

Reply:
Where can I find the corresponding public key?

Reply:
Okay, the private key can be imported and then I can export the public key...

Reply:
> Also this fixture is probably worth to be session scoped.

Is there any reason for this?